src/core/runtime.d

```diff
@@ -300,6 +301,30 @@ struct Runtime
      * value of this routine indicates to the runtime whether the tests ran
      * without error.
      *
+     * There are two options for handlers. The `bool` version is deprecated but
```

Comment:
Shouldn't this say "schedule for deprecation" unless `deprecated` is added to the deprecated overload?